mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -60,16 +60,11 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
     streams = StreamsFile(streams_filename, streamsdir=indir)
 
     # Note: input file, not a mesh file because we need dycore specific fields
-    # such as refBottomDepth and namelist fields such as config_density0 that
-    # are not guaranteed to be in the mesh file.
-    # attempt to find fields in path-local restart file but default to input
-    # file if restart is unavailable.  This is done to make analysis robust
-    # to being run once the run directory is moved to a new machine.
-    try:
-      inputfile = streams.readpath('restart')[0]
-    except ValueError, e:
-      print e, ' Trying to get mesh from input file.'
-      inputfile = streams.readpath('input')[0]
+    # such as refBottomDepth and namelist fields such as config_density0, as
+    # well as simulationStartTime, that are not guaranteed to be in the mesh file.
+    inputfile = streams.readpath('restart')[0]
+    if inputfile == None:
```

Comment:
Hi @milenaveneziani, this code won't quite work as you expect.  I think `streams.readpath('restart')` will return an empty list `[]`, which means that the previous line will already raise an exception of no restart files are found (but it won't be a very clear one).  Could you change this to something like:
```
restartFileNames = streams.readpath('restart')
if len(restartFileNames) > 0:
    inputfile = restarFileNames[0]
else:
    raise IOError("No MPAS-O restart file found: need at least one rst file for OHC calculation")
```
Note, I'd suggest making this an `IOError` or a `RuntimeError` exception because it doesn't really fit the description of a `ValueError`:
https://www.tutorialspoint.com/python/standard_exceptions.htm